package.json

```diff
@@ -4,7 +4,9 @@
   "description": "> Project description",
   "main": ".eslintrc.js",
   "scripts": {
-    "test": "jest"
+    "test": "jest",
```

Comment:
Suggestion - consider also adding a script to run Nodemon

Reply:
Will update with a nodemon script later - not doing much any work at the moment.